<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UMLEntityAttributeDialog</class>
 <widget class="QWidget" name="UMLEntityAttributeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="3">
    <widget class="QLineEdit" name="defaultValueLE"/>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="QCheckBox" name="allowNullCB">
     <property name="text">
      <string>Allow Null</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>General Properties</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="nameLE"/>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Lenght/Values:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Indexing</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QRadioButton" name="notIndexedRB">
        <property name="text">
         <string>Not Inde&amp;xed</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="indexedRB">
        <property name="text">
         <string>Indexed</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Default Value:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="UMLDatatypeWidget" name="dataTypeWidget" native="true"/>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Attributes:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2" colspan="2">
    <widget class="QComboBox" name="attributeTypesCB"/>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QCheckBox" name="autoIncrementCB">
     <property name="text">
      <string>Auto Increment:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="UMLStereotypeWidget" name="stereotypeWidget" native="true"/>
   </item>
   <item row="6" column="3">
    <widget class="QLineEdit" name="lenghtValuesLE"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UMLDatatypeWidget</class>
   <extends>QWidget</extends>
   <header>widgets/umldatatypewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UMLStereotypeWidget</class>
   <extends>QWidget</extends>
   <header>widgets/umlstereotypewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
